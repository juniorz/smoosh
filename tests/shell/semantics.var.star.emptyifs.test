IFS=""
bee="b  e   e"
set a "$bee" c

printf '<%s>\n' $*
