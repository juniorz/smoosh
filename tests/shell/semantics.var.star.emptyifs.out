<a>
<b  e   e>
<c>
